recipes/sqlitecpp/all/conanfile.py

```diff
@@ -19,11 +19,13 @@ class SQLiteCppConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "lint": [True, False, "deprecated"],
+        "omit_load_extension": [True, False],
```

Comment:
https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#recommended-feature-options-names

it could match more with our conan convention